downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md

```diff
@@ -49,39 +49,39 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
     public function main() {
         json & readonly v = {};
         
-        string|error r = v; // compilation error now
-        Employee|string s = v.cloneReadOnly(); // compilation error now
+        string|error r = v; // Compilation error now.
+        Employee|string s = v.cloneReadOnly(); // Compilation error now.
     }
     ```
 
-- Fixed a bug that previously resulted in variables that were initialized with non-isolated expressions being inferred to be `isolated` variables.
+- Fixed a bug that previously resulted in variables that were initialized with non-isolated expressions being inferred as `isolated` variables.
```

Comment:
Why can't we use "to be"?

Reply:
Since `being` is also used, the `to be` preposition causes a repetition.